packages/fast-components-react-base/src/auto-suggest/auto-suggest.tsx

```diff
@@ -87,9 +87,21 @@ class AutoSuggest extends Foundation<
     }
 
     public componentDidUpdate(prevProps: AutoSuggestProps): void {
-        const updatedMenuVisibility: boolean = this.validateMenuState(
-            this.state.isMenuOpen
-        );
+        let updatedMenuVisibility: boolean = this.state.isMenuOpen;
+
+        if (
```

Comment:
It seems to me this logic is really *part* of menu state validation - do you agree? should this logic live in `validateMenuState`?

Reply:
I was able to remove a redundant line there, but no. "validateMenuState" is a test we run every time we try to toggle to menu, it is very generic.  The logic here is specifically deciding whether we need to toggle the menu because suggestions changed in an update and does not run at any other time